### ISSUE TITLE
챌린지 상세 조회 수정 요청 건

### ISSUE THREAD

**Issue by kumakuma34** [kumakuma34/B-Side#76]
## 프론트 수정 요청 건
> 우선순위 높은 것들만 처리 할 예정 (세부 validation등 기타 스펙 다 out)
### 챌린지
- 특정 챌린지 조회 시 조회한 멤버가 챌린지에 속했는지 안속했는지 여부 
- 특정 챌린지 조회 시 해당 멤버가 그 챌린지에 글 제출 가능 상태인지 여부 

### 글
- 글 공개 / 비공개 toggle API구현 필요
- 글 제출 랭킹 조회 시 유저 프로필 썸네일 url 추가 제공

**Comment by developingbear** [comment]
글 공개 / 비공개 toggle API구현 필요 [완료]

**Comment by kumakuma34** [comment]
@developingbear 저기 To-Do 중에 문창주님이 구현 가능한거 확인한번 부탁드려요. 가져가주기 어려운거면 제가 할게요

**Comment by developingbear** [comment]
글 제출 랭킹 조회 시 유저 프로필 썸네일 url 추가 제공 -> 이거 member Id 로 프로필 url 가져오는 method 구현하면 되나요?

**Comment by kumakuma34** [comment]
> 글 제출 랭킹 조회 시 유저 프로필 썸네일 url 추가 제공 -> 이거 member Id 로 프로필 url 가져오는 method 구현하면 되나요?

네 맞습니다. 지금 멤버 이름, 글 작성 갯수(?) 이렇게 두가지 제공하고있는데 거기에 멤버 프로필 url추가해서 3개 항목 responese던져주시면됩니다
